js/components/thumbnailContainer.js

```diff
@@ -207,25 +207,21 @@ var ThumbnailContainer = React.createClass({
   },
 
   render: function() {
-    var thumbnail = Utils.findThumbnail(this.props.thumbnails, this.props.hoverTime, this.props.duration, this.props.videoVr);
     var time = isFinite(parseInt(this.props.hoverTime)) ? Utils.formatSeconds(parseInt(this.props.hoverTime)) : null;
+    var thumbnail = null;
 
+    var thumbnails = Utils.findThumbnail(this.props.thumbnails, this.props.hoverTime, this.props.duration, this.props.videoVr);
```

Comment:
Nice job finding a way to remove duplicate code!

Reply:
Thank you :-)